lte/gateway/c/core/oai/tasks/amf/amf_cn.cpp

```diff
@@ -72,6 +72,27 @@ static int amf_cn_authentication_res(amf_cn_auth_res_t* const msg) {
   OAILOG_FUNC_RETURN(LOG_NAS_AMF, rc);
 }
 
+//------------------------------------------------------------------------------
+static int amf_cn_implicit_deregister_ue(const amf_ue_ngap_id_t ue_id) {
+  int rc = RETURNok;
+  struct amf_context_s* amf_ctx_p = NULL;
```

Comment:
amf_ctx_p is not used

Reply:
Code has been modified, now `amf_ctx_p` is being used.